src/content/updates/migration/designers.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [IBM Design Language](#ibm-design-language)
+- [Design kit](#design-kit)
+- [Themes](#themes)
+- [Icons](#icons)
+- [Typography](#typography)
+- [Motion](#motion)
+- [Layout and grid](#layout-and-grid)
+- [Design migration strategy](#design-migration-strategy)
+<!-- - [Design FAQ](#design-faq) -->
+- [Feedback](#feedback)
+
+</AnchorLinks>
+
+## Resources
+
+<GridWrapper col_lg="8" flex="true" bleed="true">
+  <ClickableTile
+    title="Subscribe to Carbon Design Kit Library (Sketch Cloud)"
+    href="sketch://add-library/cloud/JaVzz"
+    type="resource">
+
+![](/images/sketch-icon.png)
+
+  </ClickableTile>
+
+  <ClickableTile
+    title="Download Carbon Design Kit Sketch file (GitHub)"
+    href="https://github.com/ibm/carbon-design-kit"
+    type="resource">
+
+![](/images/sketch-icon.png)
+</ClickableTile>
+
+<ClickableTile
+    title="Subscribe to IBM Design Language Library (Sketch Cloud)"
+    href="sketch://add-library/cloud/75VZZ"
+    type="resource">
+
+![](/images/sketch-icon.png)
+
+  </ClickableTile>
+
+</GridWrapper>
+
+## IBM Design Language
+
+Carbon X delivers the new <a href="https://www.ibm.com/design/language/" target="_blank">IBM Design Language</a> to product teams. It represents a complete visual refresh of the system, delivering the ethos of IBM Design. With tooling and  guidance on color, layout, typography, iconography, motion, patterns, and content, Carbon X is a modern, open-source framework for building digital products and experiences.
+
+## Design kit
+
+Carbon X includes a completely revised Sketch design kit, updated for the new IBM Design Language. All components in the kit have been aligned with the Design Language, as well as colors, icons, typography, motion, and grid elements. 
+
+Please note that upgrading to the v10 kit will not automatically change any files created in previous versions of the kit. In Sketch, the migration of existing v9 layouts will be a manual process for the designer.
+
+### What's changed from the v9 kit
+
+- New IBM Design Language styling
+- Improved kit structure
+- Updated text styles
+- Updated spacers
+- Icon updates
+- New color tokens
+- Updated elements page
+- New Getting Started page
+- Updated layer styles
+
+For more details on these changes to the design kit, see the What's New page in the <a href="https://github.com/ibm/carbon-design-kit" target="_blank">Design Kit Sketch file</a>.
+
+### Setting up the kit
+
+
+The preferred method for using the kit is via Sketch cloud subscription. Click the tile above to subscribe to the Carbon Design Kit as a library. Sketch cloud integration ensures that all updates to the design kit will be automatically pushed to you via notifications in the Sketch app. 
```

Comment:
Is 'subscription/subscribe' the right terminology here?

Reply:
It's the terminology that Sketch uses. . . https://blog.sketchapp.com/prototyping-libraries-on-sketch-cloud-and-an-official-ios-ui-kit-in-sketch-49-bf090c70796c